api/inventory/v1/agents.proto

```diff
@@ -723,6 +706,57 @@ message AddAgentResponse {
   int32 table_count = 20;
```

Comment:
I'm wondering why do we need this field, can we include it to one of exporter messages? I believe that it's related to MySQLdExporter

Reply:
Moved to MySQLdExporter.